src/render/tutorials/caching-http-requests/2_0.html.md

```diff
@@ -0,0 +1,337 @@
+---
+title: Caching HTTP requests
+author: oskarjerzyk
+keywords: tutorial
+date: 2019-10-21
+layout: tutorial
+knotxVersions:
+    - 2.0.0
+---
+# Overview
+In this tutorial we will add ability to cache http request to project based on [Knot.x Starter Kit](https://github.com/Knotx/knotx-starter-kit) 
+template.
+
+We will use existing `ActionFactory` from [knotx-fragments](https://github.com/Knotx/knotx-fragments) to achieve our goal
+which will be implementing custom endpoint with cache ability.
+
+## Prerequisites
+- JDK 8
+- Docker
+
+## What you're going to learn:
+- How to add cache ability to API
+- How to set up Wiremock (we don't need to implement real API, mock API will be enough for our training purposes)
+- How to deploy project to Docker Swarm
```

Comment:
We can remove this, it is not tutorial about Docker Swarm. However, we can mention about Docker Swarm in `Prerequisites`.

Reply:
fixed